src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs

```diff
@@ -146,7 +149,7 @@ private static void PrepareFields(Context? context)
 				{
 					_cursorDrawableField = _editorField.Get(editText)?.Class.GetDeclaredField("mCursorDrawable");
 				}
-				else
+				else if((int)Build.VERSION.SdkInt < 29)
 				{
 					// set differently in Android P (API 28) and higher
```

Comment:
I think this comment is no longer correct. This if branch is reachable only for API level 28 exactly.

Reply:
That's true. We'll update the `if` case and add a comment what happens with APIs 29+

Reply:
Done.